src/uu/ls/src/ls.rs

```diff
@@ -16,6 +16,7 @@ extern crate uucore;
 mod quoting_style;
 mod version_cmp;
 
+use cached::proc_macro::cached;
```

Comment:
please fix the warning :)

Reply:
I've fixed the warning - though I see failure in a CI/CD build for MinRustV: https://github.com/uutils/coreutils/pull/2083/checks?check_run_id=2373934810

Any suggestions on how to handle that would be welcome.
Maybe importing the cached just for caching uid2usr and gid2grp can be avoided and we could get away with something simpler - but I did not want to complicate things too much and this seemed like a textbook case for drop-in use of cached

Reply:
I think MinRustV fails because `cached` depends on `async-mutex` which seems to be using a feature added in Rust 1.41 (our current MSRV is 1.40). Here is the PR for that feature: https://github.com/rust-lang/rust/pull/64325/
```
Error:    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/async-mutex-1.4.0/src/lib.rs:292:31
    |
292 |     pub fn try_lock_arc(self: &Arc<Self>) -> Option<MutexGuardArc<T>> {
    |                               ^^^^^^^^^^
    |
    = note: for more information, see https://github.com/rust-lang/rust/issues/44874
    = help: consider changing to `self`, `&self`, `&mut self`, `self: Box<Self>`, `self: Rc<Self>`, `self: Arc<Self>`, or `self: Pin<P>` (where P is one of the previous types except `Self`)
```